templates/zerver/apps.html

```diff
@@ -56,6 +56,9 @@ <h2>Apps for every platform.</h2>
             <a class="" href="/apps/ios">
                 <i class="icon fa fa-mobile-phone" data-label="iOS"></i>
             </a>
+            <a href="https://github.com/zulip/zulip-terminal#zulip-terminal---zulips-official-terminal-client">
```

Comment:
We can also link to https://github.com/zulip/zulip-terminal#installation alternatively, but I like the zt's screenshot in the current link.

Reply:
Yeah, I think the screenshot is a good place to land folks at this stage of development.